### DISCUSSION TITLE
Raising Awareness: The Importance of Proper Customer Service for Students and Developers on GitHub

### DISCUSSION THREAD

**Discussion by F0XN0T** [community/community#51396] [GitHub Education]
Attention all students and developers! Have you ever faced a dismissive response from a customer support team when reaching out for assistance with a serious issue? Unfortunately, I recently experienced this with GitHub Support, and I wanted to share my story to raise awareness about the importance of proper customer service for those trying to succeed in their fields.

I reached out to GitHub Support for assistance with a crucial issue, only to receive a response that directed me to the community and education sections. As a student or developer, receiving such a response can be extremely frustrating and can hinder your progress and success on the platform.

It is crucial for companies like GitHub to take their customers' concerns seriously and provide them with timely and professional solutions. As a community, we need to demand better customer service from companies and hold them accountable for their actions.

Have you faced similar issues with customer support teams? How did you handle the situation? Do you have any tips or recommendations for others who may be struggling with similar issues? Let's start a discussion and work together to push for better treatment and support for those trying to succeed in their fields.

Thank you for your attention to this matter.

**Comment by Arison7** [comment]
I have faced the same issue my question went totally ignored, with the ticket getting closed after a week without any conversation starting. 
It's really frustrating especially given that I regarded the renewal of my pack, which I have to do within a certain amount of time.

**Reply by F0XN0T** [reply]
As a consumer and fan of Github, I understand how frustrating it can be when your support tickets are ignored, especially when it concerns the renewal of your pack. It's essential to get timely responses and guidance to avoid any disruptions in your workflow.

It's crucial for companies like Github to prioritize customer support and ensure that all issues are addressed promptly. I recommend reaching out to their support team again, using clear and concise language, and providing as much information as possible to help them understand the issue. Additionally, you can check their online community forums for answers to similar issues or seek help from other users.

As a student learning to contribute to the economy, it's essential to have access to reliable tools and platforms like Github. Hence, it's imperative for companies like Github to provide adequate support and maintain a high level of customer satisfaction to keep users loyal and foster growth.
